### PR TITLE
Installable package

### PR DESCRIPTION
**Pull request by jvanderleeuw**
Tried to make into an installable package

### CODE REVIEW

File: base-image/entrypoint.py [hunk 1]
```diff
@@ -56,10 +58,7 @@ def load_metadata(metadata_path: str) -> dict:
             won't contain metadata but will contain an error message under an 'error' key.
         """
         try:
-            if os.path.splitext(metadata_path)[1] in ('.yml', '.yaml'):
```
Comment (Chris-Schnaufer) [inline base-image/entrypoint.py]: This is the correct code. It looks like you didn't update your local copy before branching and are reverting to an older version

File: base-image/entrypoint.py [hunk 2]
```diff
@@ -7,17 +7,18 @@
 import json
 import logging
 from typing import Optional
-import yaml
```
Comment (Chris-Schnaufer) [inline base-image/entrypoint.py]: This is the correct code. It looks like you didn't update your local copy before branching and are reverting to an older version

File: base-image/entrypoint.py [hunk 3]
```diff
@@ -186,15 +186,13 @@ def parse_continue_result(result) -> tuple:
             A tuple containing the result code and result message. One or both of these
             values in the tuple may be None
         Notes:
-            A string parameter will always return a result code of None and message
+            A string parameter will always return a result code of None and message of None indicating
             of None indicating the caller needs to decide what to do.
```
Comment (Chris-Schnaufer) [inline base-image/entrypoint.py]: The start of this line repeats the above line.

**Review by Chris-Schnaufer — COMMENTED** [review]
There are still structural issues of this PR that need to be addressed.

File: base-image/entrypoint.py [hunk 4]
```diff
@@ -57,7 +59,7 @@ def load_metadata(metadata_path: str) -> dict:
         """
         try:
             if os.path.splitext(metadata_path)[1] in ('.yml', '.yaml'):
-                load_func = yaml.safe_load
+                load_func = yaml.safe_load()
```
Comment (Chris-Schnaufer) [inline base-image/entrypoint.py]: Why did you add parends? Why are you making this harder than it needs to be by changing code that's working and accepted, and isn't causing any problems?

File: setup.py [hunk 5]
```diff
@@ -0,0 +1,25 @@
+"""This script is used in order to make this package installable.
```
Comment (Chris-Schnaufer) [inline setup.py]: Why isn't this filled out properly?

**Review by Chris-Schnaufer — COMMENTED** [review]
You have too many __init__.py files. Please clean them up

**Review by Chris-Schnaufer — COMMENTED** [review]
Not ready for a PR. Please clean up

**Comment by jvanderleeuw** [comment]
This branch is unnecessary. The installable package was instead handled in agpypeline: https://github.com/AgPipeline/agpypeline
